README.md

```diff
@@ -7,5 +7,14 @@
 # Test-Plan Assets
 
 Welcome to the Octarine Test-Plane Assets. In this repository, you will find the needed assets to help demonstrate Octarine's guardrails and runtime capabilities.
+## About Octarine 
+Octarine is a cloud-native network security platform for the complete lifecycle of Kubernetes applications. It helps protect your cloud-native apps from build to runtime:,
+
+**Continuous compliance**. Meeting compliance and security obligations is not optional, but Octarine makes it easy. The platform prevents non-compliant apps from being introduced, and detects exploits and remediates in real-time for apps in production.
```

Comment:
'and detects exploits and remediates in real time for apps' - this should probably be reworded to something like 'and detects configuration vulnerabilities in real time for apps'

Reply:
This is a content provided by M&M  🤯